contrib/container-chroot-agent/entrypoint-agent.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+set -x
+
+# mount FSs from the host VM to /chroot folder
+for i in proc sys run dev; do mkdir -p /chroot/$i; done
+mount --types proc /rootproc /chroot/proc
+mount --rbind /rootsys /chroot/sys
+mount --make-rslave /chroot/sys
+mount --rbind /rootdev /chroot/dev
+mount --make-rslave /chroot/dev
```

Comment:
So far in testing docker inventory it seems these `r` (recursive) mount commands are not needed. Will continue testing to see if any issues arise.

```suggestion
mount --bind /rootsys /chroot/sys
mount --make-slave /chroot/sys
mount --bind /rootdev /chroot/dev
mount --make-slave /chroot/dev
```